readme.md

```diff
@@ -101,6 +101,50 @@ feeds
   .catch((err) => console.log(err));
 ```
 
+## Paginate back through previously published items
+
+### Definition
+
+```js
+feeds.paginate(feedId: string, options: ?PaginateOptions)
+```
+
+#### Arguments
+
+* `feedId`: The feed ID to fetch items from.
+* `options`: An object with the following keys
+  * `cursor: ?number`: the ID of the first item in the page – if not provided,
+    retrieves the most recently published items
+  * `limit: ?number`: the number of items to retrieve
```

Comment:
The default limit is (pretty arbitrarily) set to `50`. I guess it probably makes sense to document that here, will update!